packages/block-editor/src/components/url-input/index.js

```diff
@@ -212,6 +217,25 @@ class URLInput extends Component {
 		}
 	}
 
+	onFocus( event ) {
+		const { suggestions } = this.state;
+
+		const inputValue = event.target.value;
+
+		this.props.onFocus( inputValue );
+
+		// When opening the link editor, if there's a value present, we want to load the suggestions pane with the results for this input search value
+		// Don't re-run the suggestions on focus if there are already suggestions present (prevents searching again when tabbing between the input and buttons)
+		if (
+			inputValue &&
+			! this.props.disableSuggestions &&
+			! suggestions.length
```

Comment:
What's the default value of `suggestions`? Will it _always_ have a `.length`.

Reply:
It's an empty array.

Reply:
I've made it a more robust check for `! ( suggestions && suggestions.length )`